hs_geo_raster_resource/models.py

```diff
@@ -65,6 +65,7 @@ def create(cls, **kwargs):
                 if value_item not in value_arg_dict:
                     raise ValidationError("For coverage of type 'box' values for one or more "
                                           "bounding box limits or 'units' is missing.")
+                    # TODO: Pabitra - validate also the values of the bounding box
```

Comment:
@hyi It is not needed. There was some issues with the extraction code where this element was getting created with no data. I have now fixed the extraction code. So validation here is not needed. Will remove the TODO comment.